src/app/pages/dashboard/components/dashboard/dashboard.component.scss

```diff
@@ -89,9 +89,10 @@
     pointer-events: none;
   }
 
-  mat-card-content .header h3 {
+  mat-card-content .header h3,
+  .slot .wrapper h3 {
     @media(max-width: 767px) {
-      font-size: 20px;
+      font-size: 20px !important;
```

Comment:
So, why do we need to do this via ng-deep, why can't we have this style the component that needs it? It's really hard to see what this style applies to.
Also can't you avoid `!important` with more specificity?